client/app/telecom/telephony/billingAccount/billing/tollfreeHistory/telecom-telephony-billing-account-billing-tollfree-history.controller.js

```diff
@@ -36,26 +25,12 @@ angular.module("managerApp").controller("TelecomTelephonyBillingAccountBillingTo
                     consumption.priceValue = consumption.price ? consumption.price.value : null;
                 });
             });
+        }, function (err) {
+            self.consumptionData = [];
+            return new ToastError([$translate.instant("telephony_group_billing_tollfree_history_download_error"), (err.data && err.data.message) || ""].join(" "));
```

Comment:
This is not how `ToastError` works. It takes as arguments an error object AND a translation key, not a string